dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/maintenance/DefaultMaintenanceService.java

```diff
@@ -171,7 +171,7 @@ public boolean pruneData(DataElement dataElement) {
       return false;
     }
 
-    manager.delete(dataElement);
```

Comment:
Curious about what this used to do, did it actually delete the metadata? 🤔

Reply:
This was wrongly introduced a few PRs ago
https://github.com/dhis2/dhis2-core/commit/420326b1da4b1bd0ff4da6a8e96afdf66a871ba7#diff-10acfa7c6114045775d7939221d3d89bb04ecbf87cff67d6e35a8ca91fba9c1f
We shouldn't delete the `dataElement` here, only the attached values